ansible/roles/mongo/tasks/main.yml

```diff
@@ -28,6 +28,13 @@
     state: present
     line: After=network.target cloud-config.service
 
+- name: Configure mongo service to listen on all interfaces
+  lineinfile:
+    dest: /lib/systemd/system/mongod.service
+    regexp: '^( *bindIp=.*)$'
```

Comment:
This isn't working.  :angry:   I need to fix this before we merge.

Reply:
There are a couple other changes that I made to the mongo config file (located at /etc/mongod.conf, not /lib/systemd/system/mongod.service), so let's coordinate on those too.